services/visual-studio-app-center/visual-studio-app-center-releases-size.tester.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('8368844b to 7.98mb')
+  .get('/nock/nock/nock.json')
+  .intercept(nock =>
```

Comment:
Can we switch this to a live/real test, and then use the `isFileSize` validator on the badge message instead of an explicit size?

Reply:
I'd like to keep this mocked test as the App Center displays in `mebibytes` instead of `megabytes` but I will add a live test with the `isFileSize` validator.